### PR TITLE
Raiyan docker task2: nginx

### PR DESCRIPTION
**Pull request by raiyan-hosn**
Fixes #

- [ ] Does not depend on other PRs
- [x] Depends on other PR being merged first: #306 

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## What was done
Running react app in nginx server on docker container.
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: frontend/docker-compose.prod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '3.4'
+services:
+    nginx-todo:
+      stdin_open: true
```
Comment (musfique-m) [inline frontend/docker-compose.prod.yml:4]: Please remove this. We are not going to run our production container in interactive mode by default.

**Review by musfique-m — CHANGES_REQUESTED** [review]
1. `package.json` and `package-lock.json` should not be a part of this pr. We are not updating any packages in this pr. So this changes should not be a part of this pr.
2. move the `docker-compose.prod.yml` to the same level as `docker-compose.yml`.
3. Remove comment from prod files.

File: frontend/docker-compose.prod.yml (hunk 1 above)
Comment (raiyan-hosn) [reply frontend/docker-compose.prod.yml]: Removed.

**Comment by raiyan-hosn** [comment]
Removed package*.json and removed comments from prod files.

**Review by musfique-m — APPROVED** [review]
Looks good.

**Pull request merged**